### PR TITLE
add shivneri

### PR DESCRIPTION
**Pull request by ujjwalguptaofficial**
## Link
https://github.com/ujjwalguptaofficial/shivneri

## Checklist
* [x] - Shard is at least 30 days old.
* [x] - Shard has CI implemented.
* [x] - Shard has daily/weekly periodic builds (ideally with Crystal latest and nightly).

### CODE REVIEW

**Comment by Blacksmoke16** [comment]
Can you setup a periodic build?  See https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule.

File: README.md [hunk 1]
```diff
@@ -11,70 +11,70 @@ Search [Crystal Shards](https://crystalshards.xyz) or follow announcements [Crys
...
-* [Community](#community)
-  * [Unofficial](#unofficial)
-* [Resources](#resources)
-  * [Official Documentation Translations](#official-documentation-translations)
-* [Services and Apps](#services-and-apps)
-* [Tools](#tools)
-  * [DevOps](#devops)
-  * [Editor Plugins](#editor-plugins)
-  * [Shell Plugins](#shell-plugins)
+- [Awesome Crystal](#awesome-crystal)
```
Comment (Blacksmoke16) [inline README.md]: What was wrong with `*`?

File: README.md [hunk 2]
```diff
@@ -519,6 +519,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [grip](https://github.com/grip-framework/grip) - A microframework for building RESTful web applications, with ease and joy
  * [kemal](https://github.com/kemalcr/kemal) - Lightning Fast, Super Simple web framework. Inspired by Sinatra
  * [lucky](https://github.com/luckyframework/lucky) - Catch bugs early, forget about most performance issues, and spend more time on code instead of debugging and writing tests
+ * [Shivneri](https://github.com/ujjwalguptaofficial/shivneri) - Component based MVC web framework targeting good code structures, modularity and performance.
```
Comment (Blacksmoke16) [inline README.md]:
```suggestion
 * [Shivneri](https://github.com/ujjwalguptaofficial/shivneri) - Component based MVC web framework targeting good code structures, modularity and performance
```

**Comment by ujjwalguptaofficial** [comment]
@Blacksmoke16 i have added periodic build now.

File: README.md (hunk 1 above)
Comment (ujjwalguptaofficial) [reply README.md]: oh sorry, seems like some formatter ran & changed something. I should have taken a look  at changes before commiting :(.

Let me change those again.

Comment (ujjwalguptaofficial) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -536,6 +537,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 ## Unofficial
  * [Chinese-speaking Telegram Group](https://t.me/crystal_cn) - 来吧！TG 中文圈的朋友们！
  * [Crystal Programming Discord Server](https://discord.gg/YS7YvQy) - Unofficial Discord server dedicated to the Crystal Programming Language
+ * [Portuguese-speaking Telegram Group](https://t.me/crystalbrasil) - Bem vindos ao Crystal Brasil!
```
Comment (veelenga) [inline README.md]: I believe this is unrelated
